### PR TITLE
support eslint_d 2

### PR DESCRIPTION
**Pull request by sonnyp**
https://github.com/mantoni/eslint_d.js/issues/2#issuecomment-141712485

### CODE REVIEW

**Comment by sonnyp** [comment]
Works in background mode but I don't see any differences in lint time could be the 'poll' value?

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ In addition to the standard SublimeLinter settings, SublimeLinter-contrib-eslint
 |:------|:----------|:------------:|:-------------:|
 |eslint_d|runs "[eslint_d][eslint_d]" instead of "eslint"| | |
 
-[eslint_d][eslint_d] is a tool to reduce `eslint` startup time by running a server in the background. Now it works only with saved content of file, so probably you should set `Lint Mode` of SublimeLinter to `Load/save` or `Save only`. You must have `eslint_d` installed globally or locally with `npm install eslint_d`.
+[eslint_d][eslint_d] is a tool to reduce `eslint` startup time by running a server in the background. You must have `eslint_d` installed globally or locally with `npm install eslint_d`.
```
Comment (sonnyp) [inline README.md]: @roadhump you sure it works with local eslint_d ? I'm not sure to see how if it's not in the user path.

Comment (roadhump) [inline README.md]: @sonnyp by `npm_name` property, it is a functionality of `NodeLinter` class

Comment (sonnyp) [inline README.md]: :+1:

**Comment by sonnyp** [comment]
can you make a new release?

**Comment by roadhump** [comment]
done https://github.com/roadhump/SublimeLinter-eslint/releases/tag/1.8.0

it seems still not possible to make it works as `eslint` itself on unsaved files :frowning: . I'll try later this week.

**Comment by mantoni** [comment]
@roadhump Thanks! I guess you would need support to lint stdin, right? That needs a little more work on eslint_d. Any help on this would be greatly appreciated.

**Comment by sonnyp** [comment]
I'd happily send a PR to eslint_d. @roadhump what would we need?

**Comment by mantoni** [comment]
@sonnyp :+1:

**Comment by roadhump** [comment]
@sonnyp We need support of `--stdin` and `--stdin-filename` options of eslint.

**Comment by roadhump** [comment]
Ok, I've made a decision to move `eslint_d` to separate SublimeLinter plugin here https://github.com/roadhump/SublimeLinter-contrib-eslint_d. It seems SublimeLinter is not intended to run two different programs from one plugin and it can create a huge mess with current eslint plugin. I'll ready to sync relevant code of `eslint` and `eslint_d` plugins, or probably find a way to share it somehow.

I've created an issue to add `eslint_d` plugin to SublimeLinter packages and I will publish version of this plugin without `eslint_d` as soon as `eslint_d` plugin is published.

**Pull request merged**